webserver_test.py

```diff
@@ -1,15 +1,28 @@
 import re
 from subprocess import Popen
 from subprocess import PIPE
-import socket
 import sys
```

Comment:
In general, to follow good Python conventions, we'd want to rewrite this whole script as a set of classes, but we're low on time.